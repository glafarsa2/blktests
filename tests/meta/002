#!/bin/bash
# SPDX-License-Identifier: GPL-3.0+
# Copyright (C) 2017 Omar Sandoval
#
# Test a passing device test.

. tests/meta/rc

DESCRIPTION="do nothing"

test_device() {
	echo "Running ${TEST_NAME}"
	echo "Test complete"
}
