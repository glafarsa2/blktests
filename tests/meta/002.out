Running meta/002
Test complete
